src/pages/DoughStaking.svelte

```diff
@@ -296,9 +303,16 @@
                     }
                   }, 1000);
 
-                  approveToken($eth)
+                  let shouldResetAllowance = false;
+                  if(stakeAmount.bn.isGreaterThan($allowances[allowanceKey]) && $allowances[allowanceKey] > 0) {
```

Comment:
Condition 1 is BN comparison 
Condition 2 is a `number` comparison

Just want to flag this as well. Checking that the right operators are being used here, i know you can pass a number as a comparison to a BN, but then I'm wondering if the label value is being used.

Reply:
good spotted, changing it.

Reply:
+1